src/main/java/com/base64/d/main.java

```diff
@@ -1,232 +0,0 @@
-package com.base64.d;
```

Comment:
Other than the name change, this file was changed in the following ways:
1. Remove all commented out code that is not needed.
2. Remove blank lines when there are more than one consecutive blank lines.
3. Cleanup `import` statements, by removing unused imports.